package-lock.json

```
@@ -37,6 +37,8 @@
                 "fetch-cookie": "^3.1.0",
                 "fzf": "^0.5.2",
                 "globals": "^16.5.0",
+                "highlight-words": "2.0.0",
+                "globals": "^16.5.0",
```

```suggestion
                "globals": "^16.5.0",
                "highlight-words": "2.0.0",
```
globals is duplicated here. Just run `npm i` to fix it.